mirage/config.js

```diff
@@ -40,6 +40,7 @@ export default function() {
   */
   this.passthrough('https://labs-layers-api.herokuapp.com/**');
   this.passthrough('https://labs-layers-api-staging.herokuapp.com/v1/base/style.json');
+  this.passthrough('http://localhost:3000/**');
```

Comment:
Was this to bypass Mirage? If so, the proper substitute for this is to set Mirage to `false` in the `config/environment.js` file:

![image](https://user-images.githubusercontent.com/3311663/128571976-a3dcee35-e9cb-46e0-b3e6-ec8d644fab1f.png)